src/app/shared/components/common/modal-dialog/modal-dialog.component.ts

```diff
@@ -67,6 +70,11 @@ export class ModalDialogComponent<T> {
     const size = this.options?.size ? this.options.size : undefined;
 
     this.ngbModalRef = this.ngbModal.open(this.modalDialogTemplate, { size });
+
+    this.shown$ = this.ngbModalRef.shown.pipe(
```

Comment:
Is this used anywhere? Looks like dead code.

Reply:
It is used in the modal-dialog-link component as a condition for the content projection.